src/webpack/shared.js

```diff
@@ -17,5 +17,13 @@ module.exports = (babelrc) => ({
         ]
       }
     }]
+  }, {
+    test: /\.(jpe?g|png|svg|woff(2)?)$/,
```

Comment:
This test could be modified from the project couldn't it. Maybe .ico for favicons as a default?